app/uk/gov/hmrc/apisubscriptionfields/controller/FieldsDefinitionController.scala

```diff
@@ -49,8 +49,8 @@ class FieldsDefinitionController @Inject() (service: FieldsDefinitionService) ex
 
   def getAllFieldsDefinitions: Action[AnyContent] = Action.async { implicit request =>
     Logger.debug(s"[getAllFieldsDefinitions]")
-    service.getAll.map(list =>
-      Ok(Json.toJson(list))
+    service.getAll.map(fieldsDefinitionResponse =>
```

Comment:
write in a single line:
```
service.getAll.map(Ok(Json.toJson(_)))
```

Reply:
suggested code does not work - probably needs a type annotation to make it work - so I would suggest original style is OK